src/store/index.ts

```diff
@@ -0,0 +1,15 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
```

Comment:
Does not look like you use Vuex. Is this just here because you suspect future tutorials might need them?

Reply:
As said in my comment below. I plan on making more complex tutorials and I think vuex might be required for that. In my experience it's easier to remove it from an existing project than to add it later, so I selected the option when creating the project.

Reply:
True.